docs/src/doc/dev/adr/0007-css-base.stories.mdx

```diff
@@ -0,0 +1,44 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Documentation|dev/ADRs/7) CSS base" />
+
+# 7) CSS base
+
+Date: 2020-07-20
+
+## Status
+
+Accepted
+
+## Context
+
+We are in the process of implementing a component library. Those components are supposed to become portable manifestations of our Design System decisions which can get integrated into existing applications where needed.
+
+This poses a challenge when users look at our components with user agents of different default styles, and when the applications - in which the components get embedded to - contain other styles (which were not developed with isolation in mind) already. We also want the developer experience during component development to be intuitive (no surprising base styles).
+
+All component (style) development will happen on this foundation so it is important to make our component library robust and easy to contribute to.
+
+## Decision
+
+We decided that - even though inconsistent between user agent stylesheets are less of an issue today[0] than they were in the past - the need to address this challenge remains and does so early in the process (before first styles are developed) is important to avoid bug reports and double effort.
+
+We also decided that - while the definition of "intuitive" base style is arguably subjective - there is the need to have a base and its values have to be chosen carefully. While e.g. a reset of the margins of elements can help to ensure that the desired behavior correctly gets specified inside the component, un-styling the bold text of a `<strong>` tag may be unintuitive and cause confusion.
+
+The decision was to employ best practices as seen in other popular open source software, e.g. [vuetify] and [bootstrap-vue], and - as documented there - go with a solution "to correct inconsistencies across browsers and devices while providing slightly more opinionated resets to common HTML elements".
+
+We also decided to trust the task of "mitigating styles inherited from other greedy selectors" to the individual users of our library. _If_ it is needed, then the applications will know best which concrete styles need compensation.
+
+## Consequences
+
+We have to decide for one of the existing technical solutions for this challenge, e.g. [ress], [normalize.css] or [bootstrap reboot].
+
+Component development can happen against a stable base style. We must create a shared "namespace"/selector for our components in order to ensure that we apply the base styles only to the intended namespace under our control.
```

Comment:
This, in my eyes, implies two new sub tasks for the component library
* (hard requirement) employ prefixwrap or comparable - to build a shared namespace for all CSS
* (advisable, but maybe later) employ purgecss or comparable - to eliminate base CSS which has no use in the mark-up of our components